sky/setup_files/setup.py

```diff
@@ -59,7 +59,9 @@ def parse_footnote(readme: str) -> str:
     # NOTE: ray 2.0.1 requires click<=8.0.4,>=7.0; We disable the
     # shell completion for click<8.0 for backward compatibility.
     'click<=8.0.4,>=7.0',
-    'colorama',
+    # NOTE: required by awscli. To avoid ray automatically installing
+    # the latest version.
+    'colorama<0.4.5',
```

Comment:
> After this fix, the user may need to install again with pip install . to make the dependency take effect.

Just so I understand this correctly - only users running Ray > 2.0 will need to do this, right?

Reply:
Oh, only the user who manually upgrades `ray` in their environment to `>2.0` may need to do this. Otherwise, no other actions need to be taken.

Reply:
Sounds good!